docs/retries.rst

```diff
@@ -55,6 +55,8 @@ The arguments of the retries tool are listed below.
 | prefix_name                                     | String determining prefix of model names such that models are named |
 |                                                 | {prefix_name}_retries_run2 for instance.                            |
 +-------------------------------------------------+---------------------------------------------------------------------+
+| rng                                             | A random number generator or seed to use for random sampling.
```

Comment:
I see that we sometimes call this `rng` and sometimes `seed`. Actually we support either or. What do you think we should call it to be consistent?